### DISCUSSION TITLE
Explorer not starting after loggon

### DISCUSSION THREAD

**Discussion by abcpop** [valinet/ExplorerPatcher#333] [General]
Trying 22000.318.36.1 dxgi.dll I get a black screen after loggon . 
Only mouse cursor is visible. Need to CTRL ALT DEL and get  task manager to launch explorer.exe.
Rolling back to 22000.318.36.0 not such behaviour.  
I did not get such problem with previous releases.

Many thanks Valentin-Gabriel for your fantastic work
Serge

**Comment by valinet** [comment]
Can you test this, please?

[dxgi.zip](https://github.com/valinet/ExplorerPatcher/files/7528480/dxgi.zip)

Thanks

**Reply by tekyman007** [reply]
Same problem and the new zip worked for me

Thank you for your great work !

**Comment by abcpop** [comment]
This new dxgi.dll is OK . Explorer is starting after loggon.

Thankyou
------------------------------------------------------------------------
*From:* Valentin-Gabriel Radu ***@***.***>
*Sent:* Friday, November 12, 2021, 7:35 PM
*To:* valinet/ExplorerPatcher ***@***.***>
*Cc:* abcpop ***@***.***>, Author ***@***.***>
*Subject:* [valinet/ExplorerPatcher] Explorer not starting after loggon 
(Discussion #333)
